src/applications/ivc-champva/shared/components/fileUploads/MissingFileOverview.jsx

```diff
@@ -150,7 +150,7 @@ export default function MissingFileOverview({
   // eslint-disable-next-line no-unused-vars
   const apps =
     // Filter out any conditional pages that don't apply to this applicant
-    data.applicants.map(app => {
+    data?.applicants?.map(app => {
```

Comment:
Fixes a bug where this would break when applicants array was missing (as it now is on this form). Applicants array may exist in other forms though, so the functionality will work both ways.